media-fonts/fontawesome/fontawesome-5.10.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="Font-Awesome"
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+SRC_URI="https://github.com/FortAwesome/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="CC-BY-3.0 OFL-1.1"
+SLOT="0/5"
```

Comment:
I really wonder why this is so? Does something depend depend on "Font awesome 4" for example? Its not a showstopper, just out of curiosity.

Reply:
@juippis I think FontAwesome 4 ->5 was a major breaking change, IIRC I had some software a while ago that only worked with FA4, and FA5 would break it.